pkg/storage/swift/swift.go

```diff
@@ -348,33 +353,61 @@ func (d *driver) CreateStorage(cr *imageregistryv1.Config) error {
 		return fmt.Errorf("failed to get cluster infrastructure info: %v", err)
 	}
 
-	// Generate new container name if it wasn't provided.
-	// The name has cluster ID as a prefix plus "image-registry" suffix, which is complemented
-	// by 8 capital latin letters
-	// Example of a generated name: user-298xw-image-registry-FHEIBGDD
-	if cr.Spec.Storage.Swift.Container == "" {
-		cr.Spec.Storage.Swift.Container = generateContainerName(infra.Status.InfrastructureName)
-	}
+	generatedName := false
+	for i := 0; i < 5000; i++ {
```

Comment:
Does it make sense to declare 5000 as a constant? like "const numRetries = 5000"

Reply:
probably